vmdb/app/models/miq_provision_virt_workflow.rb

```diff
@@ -444,9 +444,7 @@ def allowed_dvs(_options = {}, hosts = nil)
         $log.info "MIQ(#{self.class.name}.allowed_dvs) Network DVS collection completed in [#{Time.now - st}] seconds"
       end
     end
-
-    hosts.each { |h| switches.merge!(@dvs_by_host[h.id]) }
-    switches
+    create_unified_pg(@dvs_by_host)
```

Comment:
@mkanoor The previous logic selected the networks based on the hosts that were eligible at the time.  This new method is ignoring that and returning all available switches regardless of the host filtering.